code/game/objects/structures/xeno.dm

```diff
@@ -888,31 +944,44 @@ TUNNEL
 	var/recharge_rate = 10
 	///Countdown to the next time we generate a jelly
 	var/nextjelly = 0
+	var/mob/living/carbon/xenomorph/creator = null
 
-/obj/structure/resin_jelly_pod/Initialize()
+/obj/structure/xeno/resin_jelly_pod/Initialize()
```

Comment:
make this signal based  on obj destruction
Best of all as a flag or bool to make adding new stuff to this simpler because these hardrefs are bad